### PR TITLE
Added item bonus effects of Runes Boots.

### PR DESCRIPTION
**Pull request by MrKeiKun**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Based on DP
```
When pure STR is 90 or higher, ATK + 10. When pure STR is 120 or higher, ATK + 10 additionally.
When pure INT is 90 or higher, MATK + 3%. When pure INT is 120 or higher, MATK + 2% additionally.
```

Also weight should be `10` not `100`

**Issues addressed:** 

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: db/re/item_db.conf [hunk 1]
```diff
@@ -31808,10 +31808,16 @@ item_db: (
...
-	Weight: 100
+	Weight: 10
 	Def: 5
 	Slots: 1
 	Loc: "EQP_SHOES"
+	Script: <"
+		if(readparam(bStr) >= 90 && readparam(bStr) <= 119) bonus bAtk, 10;
+		if(readparam(bStr) >= 120) bonus bAtk, 20;
+		if(readparam(bInt) >= 90 && readparam(bInt) <= 119) bonus bMatkRate, 3;
+		if(readparam(bInt) >= 120) bonus bMatkRate, 5;
```
Comment (Emistry) [inline db/re/item_db.conf:31819]:
```suggestion
		if (readparam(bStr) >= 90) bonus bAtk, 10;
		if (readparam(bStr) >= 120) bonus bAtk, 20;
		if (readparam(bInt) >= 90) bonus bMatkRate, 3;
		if (readparam(bInt) >= 120) bonus bMatkRate, 5;
```
effect are stackable when stats are higher...

File: db/re/item_db.conf [hunk 2]
```diff
@@ -31808,10 +31808,16 @@ item_db: (
 	Name: "Rune Boots"
 	Type: "IT_ARMOR"
 	Buy: 20
-	Weight: 100
+	Weight: 10
```
Comment (Emistry) [inline db/re/item_db.conf:31811]:
```suggestion
	Weight: 100
```
I believe the weight is correct at `100` which is `10.0`. Your changes was wrong perhaps?

File: db/re/item_db.conf (hunk 1 above)
Comment (MrKeiKun) [reply db/re/item_db.conf]: oh yeah. i forgot that part.

File: db/re/item_db.conf (hunk 2 above)
Comment (MrKeiKun) [reply db/re/item_db.conf]: oh weight is `int`

**Review by MishimaHaruna — APPROVED** [review]

**Pull request merged**